src/gui/trackersadditiondialog.cpp

```diff
@@ -74,14 +74,35 @@ TrackersAdditionDialog::~TrackersAdditionDialog()
     delete m_ui;
 }
 
-void TrackersAdditionDialog::onAccepted() const
+bool isValidEndpoint(const QStringView endpoint)
```

Comment:
This function should be placed inside anonymous namespace above `TrackersAdditionDialog` definitions.